lib/resources/xinetd.rb

```diff
@@ -53,12 +53,14 @@ def read_content(path = @conf_path)
       return @contents[path] if @contents.key?(path)
       file = inspec.file(path)
       if !file.file?
-        return skip_resource "Can't find file \"#{path}\""
+        skip_resource "Can't find file \"#{path}\""
```

Comment:
With #2307 now merged, a cleaner way to handle this would be to raise an actual Ruby exception. We have two to primary ones to choose from:

 * `Inspec::Exceptions::ResourceSkipped`: will skip the resource in the output
 * `Inspec::Exceptions::ResourceFailed`: will fail the resource in the output

One could make the argument that a failure is more appropriate here than a skip, but in this case, we should aim to keep consistency since that could be considered a breaking change. So, we can change this line to:

```ruby
raise Inspec::Exceptions::ResourceSkipped, "Can't find file: #{path}"
```

... and then you can remove the `return nil` below it since the `raise` will stop the resource from continuing on.